src/sentry/static/sentry/app/components/acl/feature.tsx

```diff
@@ -146,7 +167,7 @@ class Feature extends React.Component {
     // Override the renderDisabled function with a hook store function if there
     // is one registered for the feature.
     if (hookName) {
-      const hooks = HookStore.get(`feature-disabled:${hookName}`);
+      const hooks: [Function] = HookStore.get(`feature-disabled:${hookName}`);
```

Comment:
Heads up, `[Function]` is a tuple type. 

I think you may want `Function[]` or `Array<Function>`.

Reply:
There's only one element in the list with a type of Function, isn't tuple correct in this case?

Reply:
Actually, not 100% sure what hooks is in this case, but in the code below we only ever use the first element of the array.

Reply:
Yeah it should be a list, it just pulls the first one off.

Reply:
I based my feedback on the types from https://github.com/getsentry/sentry/pull/14160



-----

Looks like it's resolved.  👍